### DISCUSSION TITLE
Mac端图标圆角和Big sur图标规范有细微差异

### DISCUSSION THREAD

**Discussion by JUNEN234** [qier222/YesPlayMusic#1583] [Ideas]
图标的圆角与Apple提供的图标规范有细微出入，开发者是否考虑修改和调整呢？
<img width="570" alt="Screen Shot 2022-04-30 at 18 59 40" src="https://user-images.githubusercontent.com/42805991/166102800-deeed4e7-b90b-46e8-8cf0-b2210df31844.png">

<img width="570" alt="Screen Shot 2022-04-30 at 18 59 40" src="https://user-images.githubusercontent.com/42805991/166103208-74998ca6-5857-4f01-bb3b-d3b6674c14b8.png">

我自己在原图标基础上修改制作了一个图标PNG文件，
制作比较粗糙，边缘阴影直接被裁切，仅供演示和开发者参考
<img width="512" alt="Yesplay" src="https://user-images.githubusercontent.com/42805991/166102996-44712f78-a1c4-49f1-ab44-cd9d4002b2ef.png">

**Comment by qier222** [comment]
2.0会有新图标
